packages/actions/src/multicontract-interactions/stake/vote.ts

```diff
@@ -16,11 +16,11 @@ import { getElectionContract } from '../../contracts/election'
  * @returns A promise that resolves to a Hex string representing the transaction hash.
  */
 export async function vote(
-  client: WalletCeloClient,
+  clients: Required<Clients>,
```

Comment:
when wallet require must exist we wrap with the Required type

Reply:
sounds good.